apps/cli/src/__tests__/__examples__/bug-report-1.ts

```diff
@@ -73,9 +73,11 @@ const machine = createMachine(
        * This service will now type error if it
        * returns anything other than { id: string }
        */
+      // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Some errors are resolved by disabling them because we know what we're doing, like intentionally disabling `@typescript-eslint/require-await` for this `async` function which reproduces a bug report.